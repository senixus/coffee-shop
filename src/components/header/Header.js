import "./header.scss";
import slide from "../../assets/images/slider-separator.png";
import Navbar from "../navbar/Navbar";
import { withRouter, Link } from "react-router-dom";

const Header = ({ history }) => {
  return (
    <header className="background">
      <Navbar />
      <div className="background-child">
        <h1>Coffee Factory</h1>
        <img src={slide} alt="coffee" />
        <p>
          Lorem Ipsum is simply dummy text of the printing and typesetting
          industry. Lorem Ipsum has been the industry's standard dummy text ever
          since the 1500s, when an unknown printer took a galley of type and
          scrambled.
        </p>
        <Link onClick={() => history.push("/shop")}>shop here</Link>
      </div>
    </header>
  );
};

export default withRouter(Header);
