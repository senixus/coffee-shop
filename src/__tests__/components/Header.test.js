import { shallow } from "enzyme";
import React from "react";
import Header from "../../components/header/Header";

it("expect to render Header component", () => {
  const wrapper = shallow(<Header />);
  expect(wrapper.length).toBe(1);
  expect(wrapper).toMatchSnapshot();
});
