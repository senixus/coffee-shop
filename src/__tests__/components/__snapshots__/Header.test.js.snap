// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Header component 1`] = `
<header
  className="background"
>
  <Navbar />
  <div
    className="background-child"
  >
    <h1>
      Coffee Factory
    </h1>
    <img
      alt="coffee"
      src="slider-separator.png"
    />
    <p>
      Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled.
    </p>
    <Link
      to="/shop"
    >
      shop here
    </Link>
  </div>
</header>
`;
